site/app/controllers/admin/ConfigurationController.php

```diff
@@ -156,10 +117,10 @@ public function updateConfiguration() {
         if($name === 'forum_enabled') {
             if($entry == 1){
                 if($this->core->getAccess()->canI("forum.modify_category")) {
-                    $categories = ["General Questions 2", "Homework Help 2", "Quizzes 2" , "Tests 2"];
+                    $categories = ["General Questions", "Homework Help", "Quizzes" , "Tests"];
 
                     foreach ($categories as $category) {
-                        if (!$this->isValidCategories(-1, array($category))) {
+                        if (ForumUtils::isValidCategories(-1, array($category))) {
```

Comment:
Be careful with the params here, as I mentioned in my comment the first param that is required is the categories as a result from the query.

Reply:
Fixed